recipe/meta.yaml

```diff
@@ -7,23 +7,11 @@ package:
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://www.gap-system.org/pub/gap/gap48/tar.gz/gap4r8p10_2018_01_15-13_02.tar.gz
-  sha256: fc0dac32d196a825be5b4cab6a970aad58f1d16ca2870bd0381c94c85e7a9e5b
-  patches:
-    # Patches from https://github.com/sagemath/sage
-    # Set GAP_MEM
-    - patches/gap_shi.patch
```

Comment:
This patch is still needed. Otherwise gap will look for `GAP_DIR` in the source.

Reply:
That looks correct. Note that the patch does two unrelated things - it also starts GAP with `-m 24m` instead of `-m 256m`, while the current default value of that option is 128m. I suggest to drop it and stick to GAP defaults.